proposals/0002-root-signature-in-clang.md

```diff
@@ -759,8 +783,10 @@ Operands:
 * i32: ShaderRegister
 * i32: RegisterSpace
 * i32: ShaderVisibility ([D3D12_SHADER_VISIBILITY][d3d12_shader_visibility])
+* i32: Flags ([D3D12_SAMPLER_FLAGS][root_signature_version12_doc])
 
 [root_signature_versions_doc]: https://learn.microsoft.com/en-us/windows/win32/direct3d12/root-signature-version-1-1
+[root_signature_version12_doc]: https://microsoft.github.io/DirectX-Specs/d3d/VulkanOn12.html#definition-9
```

Comment:
It doesn't matter much since the reference name isn't really visible anywhere, but we might as well call this `root_signature_vulkanon12` for clarity, since this isn't actually a document about root signature 1.2 like the 1.1 doc above is.